content/doc/developer/building/index.adoc

```diff
@@ -0,0 +1,13 @@
+---
+layout: developerchapter
+wip: true
+references:
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Building+Jenkins
+  title: Building Jenkins
+- url: https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=3309681
+  title: Why Maven?
```

Comment:
Well, it does explain some of why we use Maven. Until this is made part of the plugin tutorial, a reference is still useful. Of course this is the first thing to be killed of.